app/uk/gov/hmrc/exports/services/DeclarationService.scala

```diff
@@ -73,15 +71,37 @@ class DeclarationService @Inject() (declarationRepository: DeclarationRepository
     }
   }
 
+  private def createDraft(
+    declaration: ExportsDeclaration,
+    parentId: String,
+    enhancedStatus: EnhancedStatus,
+    isAmendment: Boolean
+  ): Future[Some[(Boolean, String)]] = {
+    val associatedSubmissionId = if (isAmendment) declaration.declarationMeta.associatedSubmissionId else None
+    val declarationMeta = declaration.declarationMeta
+      .copy(
+        parentDeclarationId = Some(parentId),
+        parentDeclarationEnhancedStatus = Some(enhancedStatus),
+        status = if (isAmendment) AMENDMENT_DRAFT else DRAFT,
+        associatedSubmissionId = associatedSubmissionId
+      )
```

Comment:
Not related to this ticket, but as you are touching this section of code....
Can you set the `createdDateTime` and `updatedDateTime` fields to equal `now` when we make this copy please.
We ran across this issue yesterday when playing with the saved declaration's screen, when making a new copy of a declaration it should not retain the original's timestamp values. Thanks :slightly_smiling_face: